lib/cocina/models/dro.rb

```diff
@@ -6,6 +6,23 @@ module Cocina
   module Models
     # A digital repository object.  See https://github.com/sul-dlss-labs/taco/blob/master/maps/DRO.json
     class DRO < Dry::Struct
+      DRO_TYPES = %w[
```

Comment:
Would you consider naming this TYPES since it's already in the DRO context?

Reply:
yep, changed